ar/django_views/README.md

```diff
@@ -0,0 +1,43 @@
+# وجهات نظر جانغو -وقت الإنشاء!
+
+حان الوقت للتخلص من الشوائب التي انشأناها في الفصل الأخير! :)
+
+*view* هو المكان الذي وضعنا فيه المنطق الخاص بتطبيقنا. وسوف تطلب معلومات من `model` قمت بإنشائها من قبل وتمريرها إلى `template`. سوف ننشئ قالب في الفصل التالي. وجهات النظر هي مجرد وظائف بيثون والتي هي اكثر تعقيد قليلا من التي قمنا بكتابتها في هذا الفصل **مقدمة بيثون**.
+
+Views توضع في الملف `views.py`. وسوف نضيف *views* الخاصة بنا الى `blog/views.py`.
+
+## blog/views.py
+
+حسنا، دعونا نفتح هذا الملف، ونرى ما هو في موجود:
+
+{% filename %}blog/views.py{% endfilename %}
+
+```python
+من django.shortcuts استرد المحول
+إنشاء طرق العرض الخاصة بك هنا.
+```
```

Comment:
I think this needs to be in English? At least the import
`from django.shortcuts import render`

Reply:
LOL, You're right! I will remove that one from Crowdin, but we need to wait for rest of updates to be also translated - right now we have 78% coverage: https://crowdin.com/project/django-girls-tutorial/ar#